x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/select_rule_type/index.tsx

```diff
@@ -78,6 +78,7 @@ export const SelectRuleType: React.FC<SelectRuleTypeProps> = ({
       <EuiFlexGrid columns={4}>
         <EuiFlexItem>
           <EuiCard
+            data-test-subj="custom"
```

Comment:
Making this a little more descriptive would help future readers of both this code and the cypress tests. Something like `"customRuleType"`. Since `type = "query" | "saved_query"`, I've been referring to these in the app as "query rules" vs "machine learning rules," FYI.